specification/devopsinfrastructure/resource-manager/Microsoft.DevOpsInfrastructure/preview/2024-04-04-preview/devopsinfrastructure.json

```diff
@@ -1355,57 +1350,11 @@
       "type": "object",
       "description": "Describes Resource Quota",
       "properties": {
-        "properties": {
-          "$ref": "#/definitions/QuotaProperties",
-          "description": "The resource-specific properties for this resource.",
-          "x-ms-client-flatten": true
-        }
-      },
-      "allOf": [
-        {
-          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
-        }
-      ]
```

Comment:
earlier it was a Proxy resource, and now it is not ?

Reply:
It can't be a Proxy resource (or any other type of resource) because TypeSpec doesn't allow "name" property to not be a string, which Usages is (yes, the ARM contract for Usages is incorrect): https://armwiki.azurewebsites.net/api_contracts/UsagesAPIContract.html?q=usages